README.md

```diff
@@ -1,17 +1,26 @@
+![Diagram](https://raw.githubusercontent.com/Projeto-Jupiter/RocketPy/master/docs/static/RocketPy_Logo_Black.svg)
+
+<br/>
+<br/>
+
 [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/giovaniceotto/rocketpy/blob/master/docs/notebooks/getting_started_colab.ipynb)
 [![PyPI](https://img.shields.io/pypi/v/rocketpy?color=g)](https://pypi.org/project/rocketpy/)
 [![Documentation Status](https://readthedocs.org/projects/rocketpyalpha/badge/?version=latest)](https://docs.rocketpy.org/en/latest/?badge=latest)
 [![Build Status](https://app.travis-ci.com/Projeto-Jupiter/RocketPy.svg?branch=master)](https://app.travis-ci.com/Projeto-Jupiter/RocketPy)
-[![Chat on Discord](https://img.shields.io/discord/765037887016140840?logo=discord)](https://discord.gg/b6xYnNh)
 [![Contributors](https://img.shields.io/github/contributors/Projeto-Jupiter/rocketpy)](https://github.com/Projeto-Jupiter/RocketPy/graphs/contributors)
+[![Chat on Discord](https://img.shields.io/discord/765037887016140840?logo=discord)](https://discord.gg/b6xYnNh)
 [![LinkedIn](https://img.shields.io/badge/LinkedIn-0077B5?style=flat&logo=linkedin&logoColor=white)](https://www.linkedin.com/company/rocketpy)
 [![DOI](https://img.shields.io/badge/DOI-10.1061%2F%28ASCE%29AS.1943--5525.0001331-blue.svg)](http://dx.doi.org/10.1061/%28ASCE%29AS.1943-5525.0001331)
 
-# RocketPy
+<br/>
+<br/>
+
+# The RocketPy
```

Comment:
I really think 'The RocketPy' suits better in this case becuse we are adding the RocketPy LOGO right above the title and I thik it will be repetitive havig both "RocketPy"s. Also, adding 'the' reinforces the idea of presenting RocketPy wich is what we are doing

Reply:
I do note have an opinion on the "The" discussion honestly.
Maybe you can ask to other people in our comunity what they want to.
If no conclusion be done, I think we should simply go with the curent title, i.e. without the "the".

Reply:
@AdrianoAntongiovanni, I agree with your perspective. However, "The RocketPy" remembers me of "The Facebook" hahahah.

Reply:
Well. so we might skip some steps and call us Meta hahaha.
Speaking seariouslly, because we couldn't find an agreement, lets  keep the original name of the session